src/main/webapp/includes/scss/oc2017_styles.scss

```diff
@@ -5,7 +5,7 @@
 
 /* Load web fonts */
 @import url('//fonts.googleapis.com/css?family=Didact+Gothic|Open+Sans+Condensed:300,700|Open+Sans:400,400i,600,700,700i');
-@import url('../fonts/icomoon.eot?tmrw7y');
+@import url('../fonts/icomoon.eot?rittla');
```

Comment:
why did you have to change this?

Reply:
we replaced icomoon.eot with the newly dowloaded icomoon.eot with the added info icon, so we need to match the new files code

Reply:
oh ok, got it.